witness/README.md

```diff
@@ -1,15 +1,45 @@
-Witnessing
-============
+# Log Witnessing
 
-Witnessing is an important part of any transparent ecosystem, as it ensures that
-misbehavior on the part of logs can be prevented or detected. This is done by
-ensuring that the _checkpoints_ published by logs are _consistent_, which means 
-the log is presenting the same version of its contents to everyone and
-thus isn't carrying out a _split-view attack_.
+## Overview
 
-In terms of checkpoints, we consider witness implementations that use the
+Witnessing is an important part of log ecosystems that prevents logs from
+successfully accomplishing a _split-view attack_. A split-view attack is
+one in which the log prevents different populations with different views.
+Each of these views is append-only, and thus all clients checking consistency
+proofs are satisfied, yet the log is not fulfilling its claim of providing a
+_globally consistent_ list.
+
+For each witnessed log, a Witness will cosign log _checkpoints_ only if it
```

Comment:
Very much a nit: Here and several other places, you use the simple future tense (the witness will do this, each log will do that), which seems unnecessary.  How about just the present? 😊